src/components/socialMedia.js

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import {
+  FaInstagramSquare,
+  FaGithubSquare,
+  FaDribbbleSquare,
+  FaBehanceSquare,
+  FaLinkedin,
+} from "react-icons/fa"
+
+const social = [
+  {
+    id: 0,
+    name: "bahance",
+    icon: <FaBehanceSquare />,
+    link: "https://www.behance.net/liubovkapitul",
+    delay: "1000",
```

Comment:
What's this delay for? I imagine it's something related with animations... for that you can compute when used. for example
```js
map((item, index) => {

const itemDelay = 1000 + index * 200

// ...
}
```